### ISSUE TITLE
Why is there a difference between the prediction values of catboost regression model and actual values?

### ISSUE THREAD

**Issue by Liquidten** [catboost/catboost#2137]
I was very confused on the prediction values produced form the catboost regression model. The prediction are very low compared to the actual values. I was using MAE as the loss function and had an MAE of 0.858. Does this mean the prediction values should be adding 0.858 to the prediction values? I would really appreciate the answer. Thanks

**Comment by ek-ak** [comment]
Hello! 
MAE is calculated by formula https://catboost.ai/en/docs/concepts/loss-functions-regression#MAE. Regression models return  "ready for use" predictions,  you don't need to process them (i e to add something to then).  Big difference between predicted values and actual values could be caused by various reasons, for example overfitting, underfitting, imbalance in pool and a lot more.

**Comment by Liquidten** [comment]
Thank you I figured out the issue on my code.
